tool/tsh/options.go

```diff
@@ -114,6 +125,25 @@ var AllOptions = map[string]map[string]bool{
 	"XAuthLocation":                    map[string]bool{},
 }
 
+func asAgentForwardingMode(s string) client.AgentForwardingMode {
+	switch strings.ToLower(s) {
+	case forwardAgentTextNo:
+		return client.ForwardAgentNo
+
+	case forwardAgentTextYes:
+		return client.ForwardAgentYes
+
+	case forwardAgentTextLocal:
+		return client.ForwardAgentLocal
+
+	default:
+		log.Errorf(
+			"Invalid agent forwarding mode %q. Defaulting to %q",
```

Comment:
Same here.

Reply:
Saw this after hitting Merge, will fix in new PR